### DISCUSSION TITLE
Tycho build fails with -Dmaven.test.skip=true

### DISCUSSION THREAD

**Discussion by basilevs** [eclipse-tycho/tycho#2948] [Q&A] [answered]
```
% git clean -d -f
% mvn clean install  -Dmaven.localRepository=/tmp/fresh -Dmaven.test.skip=true
[ERROR] Failed to execute goal org.sonatype.plugins:maven-properties-plugin:0.0.2:filter-file (default) on project tycho-p2-extras-plugin: Could not filter properties file: /Users/vasiligulevich/git/tycho/tycho-extras/tycho-p2-extras-plugin/target/test-classes/baseTest.properties (No such file or directory) -> [Help 1]
```
This was found while investigating #2946. As full build can't complete either with or without `-Dmaven.test.skip=true` running integration tests locally takes a lot of work.
This is not a problem on 4.0.x.

**Comment by laeubi** [comment, marked as answer]
Please use `mvn clean install -DskipTests -T1C` for initial build of change and  then you can go into `tycho-its` module and run `mvn clean install -Pits -Dtest=<your test class>`
